test.py

```diff
@@ -0,0 +1,75 @@
+from ai import Minimax, RandomMove
+from game import Board
+
+class Test:
+    "used to let 2 AIs play against each other"
+    def __init__(self, id, mode):
+        self.id = id
+        self.mode = mode
+        self.done = False
+        self.outcome = None
+        self.board = Board()
+        if mode == 1:
```

Comment:
I would suggest using more meaningful values for the mode. When you have more than 2 it gets really confusing. And also mode is used in multiple places, so it could be easy to miss something while refactoring.